contracts/TokenFunctions.sol

```diff
@@ -204,19 +123,9 @@ contract TokenFunctions is Iupgradable {
     )
         external
         view
-        returns (uint amount)
+        returns (uint)
```

Comment:
looks like it's only been used in line 227 of ClaimRewards.sol, but you replaced it. if that's the case, it can be deprecated